internal/spec/spec.go

```diff
@@ -0,0 +1 @@
+package spec
```

Comment:
I'm guessing this was not intentional?

Reply:
it was not 😄 I moved the code and forgot to delete this.